### PR TITLE
#184581165 Integrating Coveralls code coverage 

### PR DESCRIPTION
**Pull request by wilbrord2**
#### What does this PR do?
This is for Integrating Coveralls code coverage 

#### Description of Task to be completed?

- [x] creating coverall and coverage badge
- [x] creating build on Travis ci
- [x] adding a script for coverall in package.json

#### How should this be manually tested?

- clone this repository 
- run npm coveralls
- run npm run test 

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
Story ID: [#184581165](https://www.pivotaltracker.com/story/show/184581165)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by nshutijonathan — CHANGES_REQUESTED** [review]
-Can you please rename your PR to meet the right standard?

**Review by nshutijonathan — CHANGES_REQUESTED** [review]
Also make sure to rebase

**Review by nshutijonathan — CHANGES_REQUESTED** [review]
can you  squash your commit message into into one?

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+repo_token: b5dAPmMgZHCARyJFm6QYaOXghAF7rTD9g
```
Comment (nshutijonathan) [inline .coveralls.yml:1]: the token should be loaded from the `.env` file not exposed to the public

File: .circleci/config.yml [hunk 2]
```diff
@@ -14,7 +15,14 @@ jobs:
...
+      #     name: code-coverage
+      #     command: npm run coverage
+      # - coveralls/upload
 workflows:
   test_app:
     jobs:
       - test
+notify:
+  webhooks:
+    - url: https://coveralls.io/webhook?repo_token=b5dAPmMgZHCARyJFm6QYaOXghAF7rTD9g
```
Comment (nshutijonathan) [inline .circleci/config.yml]: the token should be loaded from the .env file not exposed to the public

**Review by nshutijonathan — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (wilbrord2) [reply .circleci/config.yml]: done

**Review by nshutijonathan — CHANGES_REQUESTED** [review]
Hey one more final comment 

your commit message should be renamed to this `chore(integrations):integrating coveralls code coverage`

**Review by nshutijonathan — APPROVED** [review]
LGTM

**Review by nshutijonathan — APPROVED** [review]

**Pull request merged**